src/plugins/flexibleLayout/components/flexibleLayout.vue

```diff
@@ -232,18 +232,16 @@ export default {
             this.newFrameLocation = [containerIndex, insertFrameIndex];
         },
         addFrame(domainObject) {
-            if (this.newFrameLocation.length) {
-                let containerIndex = this.newFrameLocation[0],
-                    frameIndex = this.newFrameLocation[1],
-                    frame = new Frame(domainObject.identifier),
-                    container = this.containers[containerIndex];
+            let containerIndex = this.newFrameLocation.length ? this.newFrameLocation[0] : 0;
```

Comment:
This should likely be part of a different branch and PR? I don't think it's specific to Conditionals so if it's required to go out sooner, it could.

Reply:
I have no idea how the flex layout and the imagery changes got into this branch - this wasn't work that I did.

Reply:
OK let me figure this out.